### PR TITLE
Apigov 27393   update for edit apigeex and GitHub. Add Edit environment.

### PR DESCRIPTION
**Pull request by ColeSammonsAxway**
Udate for edit apigeex and GitHub. Add Edit environment to support that.

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | b73128c |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-425.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/edit_environment.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+2. Embedded Discovery Agent Settings: 2. Embedded Discovery Agent Settings: Edit Frequency, Team Ownership, API Discovery Filter, Additional tags, and Exclude Tags. Initiate Immediate Discovery can also be set.
+3. Embedded Traceability Agent Settings: If enabled, edit Frequency, Sampling Percentage, URL Path Allowed Patterns, Query Arguments Allowed Patterns, Query Arguments Sanitization Patterns, Request Headers Allowed Patterns, Request Headers Sanitization Patterns, Response Headers Allowed Patterns, Response Headers Sanitization Patterns, and Masking Characters. Sample All Errors can also be set.
+
+### Embedded Apigee X API Gateway environment
+
+These configuration steps are displayed only if there is Embedded Apigee X support for this  environment.
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/edit_environment.md:33]: I would suggest "These configuration steps are displayed only if an Embedded Apigee X environment is selected."

File: content/en/docs/connect_manage_environ/edit_environment.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Agent Configuration: Initiate editing by clicking **Edit Agent Configuration**. Edit Project ID, Developer Email Address, and Client Email Address. The Traceability Agent may be toggled on or off. Note any changes made will not be saved unless **Apply** is clicked. If you wish to cancel the changes, click **Cancel**.
+2. Embedded Discovery Agent Settings: Edit Frequency, Team Ownership, and Additional tags. Initiate Immediate Discovery can also be set.
+3. Embedded Traceability Agent Settings: Edit Frequency if enabled.
+
+### Embedded GitHub environment
+
+These configuration steps are displayed only if there is Embedded Github support for this  environment.
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/edit_environment.md:41]: I would suggest "These configuration steps are displayed only if an Embedded Github environment is selected."

**Review by robertengaxway — APPROVED** [review]
Added a few comments but looks good

**Comment by lbadenhop** [comment]
@ColeSammonsAxway, please review my edits. I changed the date in the header to previewed the file. The date was set for May 29th. Did you set it that far out for a reason? It will not display in the portal unless the date is current or past.

**Comment by lbadenhop** [comment]
@ColeSammonsAxway - also, did you want Edit your environment to be before Add your environment in the TOC? I think it should be after.

**Comment by ColeSammonsAxway** [comment]
> @ColeSammonsAxway - also, did you want Edit your environment to be before Add your environment in the TOC? I think it should be after.

It should be after. Apologies, I'm not very familiar with updating the docs.

**Pull request merged**